mne/tests/test_annotations.py

```diff
@@ -78,6 +78,9 @@ def windows_like_datetime(monkeypatch):
 
 def test_basics():
     """Test annotation class."""
+    pytest.importorskip("pandas")
+    import pandas as pd
```

Comment:
```suggestion
    pd = pytest.importorskip("pandas")
```
same thing elsewhere too

Reply:
Have changed for all those cases